apps/docs/docs/dev/01-intro.md

```diff
@@ -4,6 +4,20 @@ sidebar_position: 1
 
 # Introduction for Jayvee Developers
 
+## The Jayvee Manifesto
```

Comment:
I would make this it's own page and not just a section here. That way we can give some context when linking to it, e.g., that it is high level guidelines we want to follow when designing the language. Could then also be linked in all docs instead of just the dev ones.